### DISCUSSION TITLE
Different molecules look identical when drawn, is this a bug

### DISCUSSION THREAD

**Discussion by loluwot** [rdkit/rdkit#5376] [General] [answered]
Mol1: `CN(C)c1ccc([C@H]2[N@@]3C[C@@]4(c5ccccc5)C[N@@]2C[C@](c2ccccc2)(C3)[C@H]4O)cc1`
Mol2: `CN(C)c1ccc([C@H]2[N@]3C[C@@]4(c5ccccc5)C[N@]2C[C@](c2ccccc2)(C3)[C@@H]4O)cc1`

They don't canonicalize to the same smiles, but if I do:
```python
display(Chem.MolFromSmiles('CN(C)c1ccc([C@H]2[N@@]3C[C@@]4(c5ccccc5)C[N@@]2C[C@](c2ccccc2)(C3)[C@H]4O)cc1'))
display(Chem.MolFromSmiles('CN(C)c1ccc([C@H]2[N@]3C[C@@]4(c5ccccc5)C[N@]2C[C@](c2ccccc2)(C3)[C@@H]4O)cc1'))
```
they look identical. 
![image](https://user-images.githubusercontent.com/17971923/174402772-2247b7c9-acfe-499e-8e63-000a0a6f5090.png)

Is this intended behavior?

**Comment by IchiruTake** [comment, marked as answer]
Can you provide your RDKit version along with the full code for us to take a view on?

**Reply by loluwot** [reply]
RDkit version is 2022.03.3, python version is 3.10.4, running on Jupyter notebook.
Not much full code:
```python
from rdkit import Chem
from IPython.display import display
display(Chem.MolFromSmiles('CN(C)c1ccc([C@H]2[N@@]3C[C@@]4(c5ccccc5)C[N@@]2C[C@](c2ccccc2)(C3)[C@H]4O)cc1'))
display(Chem.MolFromSmiles('CN(C)c1ccc([C@H]2[N@]3C[C@@]4(c5ccccc5)C[N@]2C[C@](c2ccccc2)(C3)[C@@H]4O)cc1'))
```

**Reply by IchiruTake** [reply]
Chirality is hard to be visual in 2D figure, the position as you can see is just relative (overlap can occure) so I believe there are no issue. You may want to do santize and preprocess it first. 
Normally you can try MolDraw2DCairo to have extended option

**Reply by loluwot** [reply]
Ok, thank you

**Reply by IchiruTake** [reply]
I found the issue. It turns out the display does not support multi drawing instead.
According to your result, IPython display exactly the same within the same kernel (run on GG Colab)
To resolve this, you must use IPythonConsole in Chem.Draw which supported it.
You may also require to split out the drawing of two mols into two different kernels instead.
In general, the RDKit code worked correctly, not the bug. The bus is on IPython.

![image](https://user-images.githubusercontent.com/66363520/174710813-3af5b541-65fc-4153-af7c-ec5de6c0c373.png)

![image](https://user-images.githubusercontent.com/66363520/174710869-079be670-6275-47a5-be3c-4129751fb738.png)

**Reply by IchiruTake** [reply]
> Ok, thank you

You can see this link to help you: https://www.rdkit.org/docs/Cookbook.html in Drawing Molecules (Jupyter)
